aero.minova.core.application.system.service/src/main/java/aero/minova/core/application/system/ControllerExceptionHandler.java

```diff
@@ -213,12 +213,20 @@ public void saveErrorInDatabase(Exception e) {
 			callableErrorStatement.setString(1, username);
 			callableErrorStatement.setString(2, e.getMessage());
 			callableErrorStatement.setTimestamp(3, timeOfError);
-			logger.info(
-					"CAS : Execute : " + errorStatement + " with values: " + username + ", " + e.getMessage() + ", " + timeOfError + "/n" + e.getStackTrace());
+			{
+				StringWriter sw = new StringWriter();
+				PrintWriter pw = new PrintWriter(sw);
+				e.printStackTrace(pw);
+				logger.info(
+						"CAS : Execute : " + errorStatement + " with values: " + username + ", " + e.getMessage() + ", " + timeOfError + "/n" + sw.toString());
+			}
 			callableErrorStatement.executeUpdate();
 			connection.commit();
 		} catch (SQLException e1) {
-			customLogger.errorLogger.error("CAS : Error could not be saved in database." + "/n" + e1.getStackTrace());
+			StringWriter sw = new StringWriter();
+			PrintWriter pw = new PrintWriter(sw);
+			e1.printStackTrace(pw);
+			customLogger.errorLogger.error("CAS : Error could not be saved in database." + "/n" + sw.toString());
```

Comment:
Man könnte den Logger-Aufruf hier auch gleich mit korrigieren.

Reply:
Was  meinst du damit?

Reply:
customLogger.logError(....)

Reply:
Hab ich aber auch schon in anderen branches korrigiert, ist also vll gar nicht so wichtig.

Reply:
Dann lasse ich es.